### PR TITLE
feat(silent): hide unmute if participant joined without audio

### PR DESCRIPTION
**Pull request by nbeckindy**
A fix for this issue I submitted earlier: https://github.com/jitsi/jitsi-meet/issues/14559

Not sure if this is the way you'd want to go about resolving this problem, but this is how we handled it.

Depends on this PR in the lib-jitsi-meet repo: https://github.com/jitsi/lib-jitsi-meet/pull/2534

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -1676,6 +1676,18 @@ export default {
                 });
             }
         );
+
+        room.on(
+            JitsiConferenceEvents.SILENT_STATUS_CHANGED,
+            (id, isSilent) => {
```
Comment (damencho) [inline conference.js:1682]: Why do we need it and here, isn't the one in `base/conference/actions.any.ts` enough?

Comment (nbeckindy) [reply conference.js:1682]: I thought the same thing, but if I remove the listener from `conference.js`, other browser-based participants don't see the silent status. And if I remove the listener in `base/conference/actions.any.ts`, mobile app participants don't see the status. I could only get it to work on both platforms with both of these listeners. I never could figure out why that's the case though.

**Comment by damencho** [comment]
Jenkins, test this please.

File: react/features/video-menu/components/native/RemoteVideoMenu.tsx [hunk 2]
```diff
@@ -258,6 +264,7 @@ function _mapStateToProps(state: IReduxState, ownProps: any) {
         _disablePrivateChat: Boolean(disablePrivateChat) || _iAmVisitor,
         _isBreakoutRoom,
         _isParticipantAvailable: Boolean(isParticipantAvailable),
+        _isParticipantSilent: Boolean(isParticipantAvailable?.isSilent),
         _moderator: moderator,
```
Comment (damencho) [inline react/features/video-menu/components/native/RemoteVideoMenu.tsx:268]: Can you rename, isParticipantAvailable to participant so it is obvious what it is :) Thanks.

**Comment by damencho** [comment]
Jenkins, test this please.

**Review by damencho — APPROVED** [review]

**Comment by damencho** [comment]
Thank you for your contribution.

**Pull request merged**